src/progressBar.js

```diff
@@ -18,7 +18,7 @@ class ProgressBarController {
     }
 
     // Don't do any of this while testing
-    if (process.env.NODE_ENV === "test") {
+    if (process.env.NODE_ENV === "lerna-test") {
```

Comment:
Yeah, wonky whenever we're not running interactively.

Maybe a better check here would be whether we're connected to a tty?

```javascript
if (process.stdout.isTTY) {
    ...
}
```